src/vmm/src/persist.rs

```diff
@@ -585,6 +597,11 @@ mod tests {
                 _ => assert!(res.is_ok()),
             }
         }
+
+        assert!(
```

Comment:
add a test case for when:
* there are more than 2 dots in the string
* only the last part of the string is incorrect: e.g: 0.21.x

Reply:
New test cases added.